src/main/flight/position.c

```diff
@@ -103,6 +106,7 @@ void calculateEstimatedAltitude(timeUs_t currentTimeUs)
 
     int32_t baroAlt = 0;
     int32_t gpsAlt = 0;
+    uint8_t gpsNumSat = 0;
```

Comment:
(better use `unsigned` here. Gcc is quite bad at handling smaller variables)

Reply:
1) There is a lot of uint8_t usages already in code
2) Previously I got from one of the reviewers advise to move from uint16_t to uint8_t.
Looks like a matter of taste. If not - I would like to see some references to code-style or something else that prohibit usage of uint8_t.

Reply:
There is difference between local variables and others.
For memory allocated variables, it is reasonable to use small data type, saving memory and expressing intent.
For function arguments, it depends. word sized variables will be slightly faster (smaller variables must be converted to 32 bit register), but sometimes it informs user about expected/available range.
Word sized local variables are faster and so it IMO better to use simple `signed`/`unsigned` (and local use means that it is easy to check), unless size is important for some reason.

The performance difference is not that big (1 instruction per use + possibly slower memory access where there is register pressure. Also compiler may possibly use more optimization, like undefined integer overflow).

Reply:
This function is called with 40HZ only - no problem to call one more instruction. IMO 16 bytes of flash is more important in this particular case.